plugins/automation/spec/requests/admin_discourse_automation_automations_spec.rb

```diff
@@ -72,6 +72,16 @@
              }
         expect(response.status).to eq(200)
       end
+
+      it "logs a staff action" do
+        expect {
+          post "/admin/plugins/automation/automations.json", params: { automation: { script: } }
+        }.to change { UserHistory.where(custom_type: "create_automation").count }.by(1)
+
+        user_history = UserHistory.last
+        expect(user_history.custom_type).to eq("create_automation")
+        expect(user_history.details).to include("script: #{script}")
```

Comment:
We could use `have_attributes`:

```suggestion
        expect(user_history).to have_attributes(
          custom_type: "create_automation",
          details: /script: #{script}/,
        )
```